test/core.jl

```diff
@@ -27,16 +28,15 @@ using Base.Test
         end
         img[2] = zero(eltype(img))
         @test A[2] == zero(eltype(A))
-        img[3] = one(eltype(img))
-        @test A[3] == one(eltype(A))
+        img[3] = oneunit(eltype(img)) # fails for RGB{N0f8} (no method for one)
```

Comment:
maybe worth filing an issue there

Reply:
I think the comment is left over from before I convinced myself that `oneunit` is the right thing to use here.  (There is no failure with `oneunit`.)  If you agree that `oneunit` is correct, I'll drop the comment.

Reply:
Yes, `oneunit` is correct. `one` is the multiplicative identity, which in this case would be just `1::Int` or similar.